modul/asteroid.py

```
@@ -2,7 +2,10 @@
 
 import math
 import random
+<<<<<<< HEAD
```

Pylint (4.0.4)

[error] Parsing failed: 'invalid syntax (modul.asteroid, line 5)'

(E0001)